lib/vhost/rte_vhost/virtio_net.c

```diff
@@ -95,7 +97,7 @@ static inline void __attribute__((always_inline))
 do_flush_shadow_used_ring(struct virtio_net *dev, struct vhost_virtqueue *vq,
 			  uint16_t to, uint16_t from, uint16_t size)
 {
-	rte_memcpy(&vq->used->ring[to],
```

Comment:
The files in rte_vhost are a temporary fork of the DPDK rte_vhost library, and we should not modify them if at all possible to make updates simpler. Also, the vhost code is inextricably tied to DPDK, so there's no reason to remove rte_ calls from it.